pyvista/core/filters/data_set.py

```diff
@@ -1097,6 +1097,66 @@ def outline_corners(  # type: ignore[misc]
         _update_alg(alg, progress_bar, 'Producing an Outline of the Corners')
         return wrap(alg.GetOutputDataObject(0))
 
+    def gaussian_splatting(  # type: ignore[misc]
+        self: _DataSetType,
+        *,
+        radius: float = 0.1,
+        progress_bar: bool = False,
+    ):
+        """Splat points into a volume using a Gaussian distribution.
+
+        This filter uses :vtk:`vtkGaussianSplatter` to splat points into a volume
+        dataset. Each point is surrounded with a Gaussian distribution function
+        weighted by input scalar data. The distribution function is volumetrically
+        sampled to create a structured dataset.
+
+        .. versionadded:: 0.46
+
+        Parameters
+        ----------
+        radius : float, default: 0.1
+            Standard deviation of the Gaussian distribution in
+            world coordinates. This determines the "width" of the
+            splatter in terms of the distribution.
```

Comment:
I don't think this is quite right, see [SetRadius](https://vtk.org/doc/nightly/html/classvtkGaussianSplatter.html#a3628fc130f401e016dc9e9a788897ef3)
```suggestion
            This value is expressed as a percentage of the length of the longest side of
            the sampling volume. Smaller numbers greatly reduce execution time.
```